resources/public/api/conf/2.0/coding_out_underpayments_retrieve.yaml

```diff
@@ -59,6 +59,8 @@ responses:
             $ref: "./common/errors.yaml#/components/examples/ruleTaxYearRangeInvalid"
           FORMAT_SOURCE:
             $ref: "./common/errors.yaml#/components/examples/formatSource"
+          RULE_INCORRECT_GOV_TEST_SCENARIO:
+            $ref: './common/errors.yaml#/components/examples/ruleIncorrectGovTestScenario'
```

Comment:
V2 ones seems like its using double quotes instead of single
Better to keep is consistent and use double quotes for v2

Reply:
Cool, will update now

Reply:
Thanks